lib/matplotlib/axes/_axes.py

```diff
@@ -3503,18 +3503,23 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
             everything is drawn horizontally.
 
         whis : float, sequence, or string (default = 1.5)
-            As a float, determines the reach of the whiskers beyond the
-            first and third quartiles. In other words, where IQR is the
-            interquartile range (`Q3-Q1`), the upper whisker will extend to
-            last datum less than `Q3 + whis*IQR`). Similarly, the lower whisker
-            will extend to the first datum greater than `Q1 - whis*IQR`.
-            Beyond the whiskers, data
-            are considered outliers and are plotted as individual
-            points. Alternatively, set
-            this to an ascending sequence of percentile (e.g., [5, 95])
-            to set the whiskers at specific percentiles of the data.
-            Finally, ``whis`` can be the string ``'range'`` to force the
-            whiskers to the min and max of the data.
+            The position of the whiskers.
+
+            If a float, the lower whisker is at position ``Q1 - whis*(Q3-Q1)``,
+            and the upper whisker at position ``Q3 + whis*(Q3-Q1)``, where Q1
+            and Q3 are the first and third quartiles.
+
+            If a pair of floats, they indicate the percentiles at which to
+            draw the whiskers (e.g., (5, 95)).  In particular, setting this to
+            [0, 100] results in whiskers covering the whole range of the data.
```

Comment:
```suggestion
            (0, 100) results in whiskers covering the whole range of the data.
```
Keep this consistent and not switch between tuples and lists. While both are possible, tuple seems more appropriate semantically.